### PR TITLE
Add YTP2CCP

### PR DESCRIPTION
**Pull request by TheAio**
Add YTP2CCP under the fun catagory.

- [x] I have read the [contribution guidelines](./CONTRIBUTING.md).
- [x] I agree to release my contributions under the [CC-BY-SA](./LICENSE.md) license.
- [ ] I have read the automated feedback from the spell-checker after submitting this pull request. 

## Summary of changes

Add YTP2CCP with pastebin link under "fun" catagory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ If you want to contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md).
 - tracc: An XM module tracker/player for ComputerCraft, supporting 8 channel polyphony in-game. **[Source](https://github.com/MCJack123/tracc/tree/playAudio)**
 - YahtCC: An implementation of the *[Yahtzee](https://en.wikipedia.org/wiki/Yahtzee)* dice game. **[Source](https://gist.github.com/MCJack123/4f7f1635998f44630c8440e81213d32e)**
 - YouCube: Access YouTube and other services in ComputerCraft. **[Source](https://github.com/Commandcracker/YouCube)**
+- YTP2CCP: Convert commented YouTube piano notes and play them in CC:T. **[pastebin](https://pastebin.com/nxEMWHY3)**
```
Comment (tomodachi94) [inline README.md]:
```suggestion
- YTP2CCP: Convert commented YouTube piano notes and play them in CC:T. **[Pastebin](https://pastebin.com/nxEMWHY3)**
```

**Review by tomodachi94 — CHANGES_REQUESTED** [review]
This looks really good except for one minor thing.

**Review by tomodachi94 — APPROVED** [review]

**Comment by tomodachi94** [comment]
@TheAio minor question, is this available on GitHub or something? I have a few minor improvements I'd like to make.

**Comment by TheAio** [comment]
@tomodachi94 unfortunately not. But you are welcome to copy the code and modify it or just make a repo out of it, however please leave the first line as some kind of credit.

**Pull request merged**